buck2/golang/README.md

```diff
@@ -27,13 +27,13 @@ The `main` cell and `library` cell:
 To test this cell with RE run (after setting up `.buckconfig` as indicated below):
 
 ```
-buck2 test //go/greeting:greeting_test
+buck2 test --target-platforms //platforms:remote_platform //go/greeting:greeting_test
```

Comment:
why are you using rbe as a target platform?

Reply:
oops, forgot to replace this one while refactoring yesterday. Addressed.